web/src/components/onboarding/OnboardingButtonGroup.tsx

```diff
@@ -33,7 +33,7 @@ export const OnboardingButtonGroup = (props: Props): ReactElement => {
 
   return (
     <div className="margin-bottom-2">
-      <ActionBarLayout reverseInMobile={true}>
+      <ActionBarLayout>
```

Comment:
[dust] Any particular reason we don't call out the value as `false` here for readability's sake?

Reply:
I don't know that I necessarily agree. I thought the whole point was that we wanted this to be default behavior unless explicitly stated otherwise?

Reply:
Pro con, Less boiler plate / mainting of things vs clarity / documentation

I'm leaning toward letting it default, less clear but I think it would hard to force people to define the prop consistently and we would end up eventually having a mix of explicit and not.

Reply:
I'm good deffering to @FarazA22 as he writes the ticket personally and we don't have precedent yet.

Reply:
The default design pattern is to reverse the ordering on mobile, if we make the prop required, it will cause us to do a gut check by reviewing the designs and AC. This could prevent design bugs when A/C deviating from default behavior is not called out. 

I think the disabling feature will be sparingly used. But I think it's something that can be easily missed in AC. Should we rely on design identifying it in their review? I prefer it to be optional.

Reply:
I think it's an odd precedent for us to require things that should be otherwise the default and intentional behavior.